packages/cursorless-vscode/src/ide/vscode/VscodeTextEditorImpl.ts

```diff
@@ -175,7 +175,7 @@ export class VscodeTextEditorImpl implements EditableTextEditor {
     await vscode.commands.executeCommand("editor.action.clipboardCopyAction");
   }
 
-  public async clipboardPaste(_ranges?: Range[]): Promise<void> {
```

Comment:
Paste uses a destination so passing a range into this function makes no sense. How are they going to handle before and after?